instrumentation/spring/spring-boot-resources/testing/build.gradle.kts

```diff
@@ -6,7 +6,7 @@ dependencies {
   testCompileOnly("com.google.auto.service:auto-service-annotations")
 
   testImplementation("org.mockito:mockito-junit-jupiter")
-  testImplementation(project(":instrumentation:spring:spring-boot-resources:library"))
+  testImplementation(project(":instrumentation:spring:spring-boot-resources:javaagent"))
```

Comment:
This breaks the test, because it expects a file [to be missing](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/3b77cc4b2de90ca821d53234e76ea8f279cfc34f/instrumentation/spring/spring-boot-resources/testing/src/test/java/io/opentelemetry/instrumentation/spring/resources/TestBootInfClassesResource.java#L26) - but it's present in the [javaagent test dir](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/11b97c152ecddea93c2f39dd7a032647897e9f30/instrumentation/spring/spring-boot-resources/javaagent/src/test/resources/application.properties#L3)

Reply:
ah, I see, maybe this `testing` module should be renamed to `javaagent-unit-tests`? https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/contributing/writing-instrumentation.md#writing-java-agent-unit-tests

Reply:
done